guacamol/common_scoring_functions.py

```diff
@@ -55,6 +55,18 @@ def score_mol(self, mol: Chem.Mol) -> float:
         fp = get_fingerprint(mol, self.fp_type)
         return TanimotoSimilarity(fp, self.ref_fp)
 
+    def __hash__(self):
```

Comment:
I don't see where hashes of the scoring functions / score modifiers are used. 

What use case do you have in mind?